### DISCUSSION TITLE
CDN try to use process and cased this file error hey-listen.es.js:3:5

### DISCUSSION THREAD

**Discussion by mortzaCFT** [WalletConnect/walletconnect-monorepo#5224] [Developer Support]
Hello guys , im new here to use web3modal and i have some problem with working CDN's with it.

Our project is just simply want to use web3modal for integrate wallet mobiles with our app, so we can gather thier wallet addresses (ONLY) for sending them rewards etc...

As i said we use CDN,  and here is the cdn's i used they working( as i expect):

```
<script src="https://unpkg.com/web3@latest/dist/web3.min.js"></script>
<script src="https://unpkg.com/web3modal"></script>
<script src="https://unpkg.com/evm-chains/lib/index.js"></script>
<script src="https://unpkg.com/@walletconnect/web3-provider"></script>
<script src="https://unpkg.com/fortmatic@2.0.6/dist/fortmatic.js"></script>
<script src="https://unpkg.com/@walletconnect/sign-client"></script>
<script type="module"> import { WalletConnectModal } from 'https://unpkg.com/@walletconnect/modal'</script>
```

The error i dealing with it is:
About this file named _hey-listen.es.js_
The error Uncaught (in promise) ReferenceError: process is not defined is occurring because the process object is not available in the browser environment, @walletconnect/modal, is trying to use the process object, which is causing the error.

I dont know how i get this error but it's lasted lib so i  dont know why i can't using this lib

The code that i just want to get result from it , throgh its official example:
<script type="module">import { WalletConnectModal } from 'https://unpkg.com/@walletconnect/modal';
  const modal = new WalletConnectModal ({
    projectId: '89ff2f66cafc43d6b57b6ac685fbc965',
    chains: ['eip155:1'],
  });
 </script>
 
 The official example doesn't work! 
 Please update them!
 With hope for sloving it..(its been 1 week!)

**Comment by glitch-txs** [comment]
Web3Modal doesn't support CDN, you can however use Vite with AppKit (perv Web3Modal) JS
https://docs.walletconnect.com/appkit/javascript/core/installation
